<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>RaptorCopyPage</class>
    <widget class="QWidget" name="RaptorCopyPage">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1008</width>
                <height>647</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>RaptorCopyPage</string>
        </property>
        <layout class="QVBoxLayout" name="_CopyPageLayout">
            <property name="spacing">
                <number>0</number>
            </property>
            <property name="leftMargin">
                <number>0</number>
            </property>
            <property name="topMargin">
                <number>0</number>
            </property>
            <property name="rightMargin">
                <number>0</number>
            </property>
            <property name="bottomMargin">
                <number>0</number>
            </property>
            <item>
                <widget class="QSplitter" name="_Splitter">
                    <property name="orientation">
                        <enum>Qt::Orientation::Horizontal</enum>
                    </property>
                    <property name="handleWidth">
                        <number>1</number>
                    </property>
                    <property name="childrenCollapsible">
                        <bool>false</bool>
                    </property>
                    <widget class="QSplitter" name="_Splittes">
                        <property name="orientation">
                            <enum>Qt::Orientation::Horizontal</enum>
                        </property>
                        <property name="handleWidth">
                            <number>1</number>
                        </property>
                        <property name="childrenCollapsible">
                            <bool>false</bool>
                        </property>
                        <widget class="RaptorTableView" name="_SourceView">
                            <property name="verticalScrollBarPolicy">
                                <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
                            </property>
                        </widget>
                        <widget class="RaptorTableView" name="_TargetView">
                            <property name="verticalScrollBarPolicy">
                                <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
                            </property>
                            <property name="selectionMode">
                                <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
                            </property>
                        </widget>
                    </widget>
                    <widget class="QWidget" name="_ItemAttributePanel" native="true">
                        <layout class="QGridLayout" name="_ItemAttributePanelLayout">
                            <property name="leftMargin">
                                <number>12</number>
                            </property>
                            <property name="topMargin">
                                <number>12</number>
                            </property>
                            <property name="rightMargin">
                                <number>0</number>
                            </property>
                            <property name="bottomMargin">
                                <number>12</number>
                            </property>
                            <property name="spacing">
                                <number>12</number>
                            </property>
                            <item row="4" column="1">
                                <widget class="QCheckBox" name="_ItemToShowAll">
                                    <property name="minimumSize">
                                        <size>
                                            <width>0</width>
                                            <height>26</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>16777215</width>
                                            <height>26</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                </widget>
                            </item>
                            <item row="6" column="1">
                                <spacer name="_ItemAttributePanelSpacer">
                                    <property name="orientation">
                                        <enum>Qt::Orientation::Vertical</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>40</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item row="5" column="1">
                                <widget class="QWidget" name="_ItemControlPanel" native="true">
                                    <layout class="QHBoxLayout" name="_ItemControlPanelLayout">
                                        <property name="spacing">
                                            <number>12</number>
                                        </property>
                                        <property name="leftMargin">
                                            <number>0</number>
                                        </property>
                                        <property name="topMargin">
                                            <number>0</number>
                                        </property>
                                        <property name="rightMargin">
                                            <number>0</number>
                                        </property>
                                        <property name="bottomMargin">
                                            <number>0</number>
                                        </property>
                                        <item>
                                            <widget class="QPushButton" name="_ItemBack">
                                                <property name="minimumSize">
                                                    <size>
                                                        <width>0</width>
                                                        <height>26</height>
                                                    </size>
                                                </property>
                                                <property name="maximumSize">
                                                    <size>
                                                        <width>16777215</width>
                                                        <height>26</height>
                                                    </size>
                                                </property>
                                                <property name="text">
                                                    <string/>
                                                </property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QPushButton" name="_ItemRoot">
                                                <property name="minimumSize">
                                                    <size>
                                                        <width>0</width>
                                                        <height>26</height>
                                                    </size>
                                                </property>
                                                <property name="maximumSize">
                                                    <size>
                                                        <width>16777215</width>
                                                        <height>26</height>
                                                    </size>
                                                </property>
                                                <property name="text">
                                                    <string/>
                                                </property>
                                            </widget>
                                        </item>
                                    </layout>
                                </widget>
                            </item>
                            <item row="3" column="1">
                                <widget class="QPushButton" name="_ItemShowLog">
                                    <property name="minimumSize">
                                        <size>
                                            <width>0</width>
                                            <height>26</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>16777215</width>
                                            <height>26</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                    <property name="checkable">
                                        <bool>true</bool>
                                    </property>
                                </widget>
                            </item>
                            <item row="2" column="0">
                                <widget class="QLabel" name="_ItemViewTip">
                                    <property name="minimumSize">
                                        <size>
                                            <width>0</width>
                                            <height>26</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>16777215</width>
                                            <height>16777215</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTop|Qt::AlignmentFlag::AlignTrailing</set>
                                    </property>
                                </widget>
                            </item>
                            <item row="2" column="1">
                                <widget class="QStackedWidget" name="_ItemViewPanel">
                                    <property name="currentIndex">
                                        <number>1</number>
                                    </property>
                                    <widget class="QWidget" name="_ItemLogPanel">
                                        <layout class="QVBoxLayout" name="_ItemLogPanelLayout">
                                            <property name="spacing">
                                                <number>12</number>
                                            </property>
                                            <property name="leftMargin">
                                                <number>0</number>
                                            </property>
                                            <property name="topMargin">
                                                <number>0</number>
                                            </property>
                                            <property name="rightMargin">
                                                <number>0</number>
                                            </property>
                                            <property name="bottomMargin">
                                                <number>0</number>
                                            </property>
                                            <item>
                                                <widget class="QTextEdit" name="_ItemLog">
                                                    <property name="readOnly">
                                                        <bool>true</bool>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QPushButton" name="_ItemClearLog">
                                                    <property name="minimumSize">
                                                        <size>
                                                            <width>0</width>
                                                            <height>26</height>
                                                        </size>
                                                    </property>
                                                    <property name="maximumSize">
                                                        <size>
                                                            <width>16777215</width>
                                                            <height>26</height>
                                                        </size>
                                                    </property>
                                                    <property name="text">
                                                        <string/>
                                                    </property>
                                                </widget>
                                            </item>
                                        </layout>
                                    </widget>
                                    <widget class="QWidget" name="_ItemQueuePanel">
                                        <layout class="QVBoxLayout" name="_ItemQueuePanelLayout">
                                            <property name="spacing">
                                                <number>12</number>
                                            </property>
                                            <property name="leftMargin">
                                                <number>0</number>
                                            </property>
                                            <property name="topMargin">
                                                <number>0</number>
                                            </property>
                                            <property name="rightMargin">
                                                <number>0</number>
                                            </property>
                                            <property name="bottomMargin">
                                                <number>0</number>
                                            </property>
                                            <item>
                                                <widget class="QTableView" name="_ItemQueueView"/>
                                            </item>
                                        </layout>
                                    </widget>
                                </widget>
                            </item>
                            <item row="0" column="0">
                                <widget class="QLabel" name="_ItemUserTip">
                                    <property name="minimumSize">
                                        <size>
                                            <width>0</width>
                                            <height>26</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>16777215</width>
                                            <height>26</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item row="0" column="1">
                                <widget class="QPushButton" name="_ItemUser">
                                    <property name="minimumSize">
                                        <size>
                                            <width>0</width>
                                            <height>26</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>16777215</width>
                                            <height>26</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                </widget>
            </item>
        </layout>
    </widget>
    <customwidgets>
        <customwidget>
            <class>RaptorTableView</class>
            <extends>QTableView</extends>
            <header>../../../Src/Component/TableView/RaptorTableView.h</header>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
